### PR TITLE
Ch slack travis integration #142465231

### PR DESCRIPTION
**Pull request by layodi**
#### What does this PR do? 
Ensures that Travis CI is integrated to Slack.

#### Description of Task to be completed? 
Send slack notifications every time Travis runs after a commit or pull request is made.

#### How should this be manually tested? 
Slack notifications sent to #andela_vulcan_updates channel

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#142465231

### CODE REVIEW

**Review by Mbarak-Mbigo — COMMENTED** [review]
It looks properly implemented

File: .gitignore [hunk 1]
```diff
@@ -62,3 +62,4 @@ target/
 # External Libraries 
 wger/core/static/bower_components
 node_modules
+wger-venv/*
```
Comment (amwaleh) [inline .gitignore]: Add this to your global .gitignore

Comment (amwaleh) [reply .gitignore]: http://islegend.com/development/setting-global-gitignore-mac-windows/

**Review by amwaleh — CHANGES_REQUESTED** [review]
Looks good!! just remove the wger-venv

**Review by amwaleh — APPROVED** [review]

**Pull request merged**